tests/testArgumentParser.py

```diff
@@ -289,13 +290,15 @@ def testAtFile(self):
     def testLogDest(self):
         """Test --logdest
         """
-        logFile = "tests_argumentParser_testLog_temp.txt"
-        self.ap.parse_args(
-            config = self.config,
-            args = [DataPath, "--logdest", logFile],
-        )
-        self.assertTrue(os.path.isfile(logFile))
-        os.remove(logFile)
```

Comment:
I just pushed a corrected version that actually test something.  Running Jenkins with this patch and without the utils patch now.